src/signals/incident/components/form/MapSelectors/Asset/Selector/DetailPanel/DetailPanel.test.tsx

```diff
@@ -563,42 +545,24 @@ describe('DetailPanel', () => {
   })
 
   it('selection nearby details', () => {
-    const selection = {
-      label: 'Huisafval',
-      description: 'Gemeld op: 01-01-1970',
-      type: NEARBY_TYPE,
-    }
-
-    render(
-      withAssetSelectContext(<DetailPanel {...props} />, {
-        ...contextValue,
-        selection,
-      })
-    )
-
-    expect(screen.getByText(selection.label)).toBeInTheDocument()
-    expect(screen.getByText(selection.description)).toBeInTheDocument()
-  })
-
-  it('sends an API request, when an object is selected on the map, to get incidents with equal coordinates', async () => {
-    const selection = {
-      label: 'Huisafval',
-      description: 'Gemeld op: 01-01-1970',
-      type: 'Rest',
-      coordinates: { lat: 1, lng: 2 },
-    }
+    const selection = [
+      {
+        id: 'bla',
+        label: 'Huisafval',
+        description: 'Gemeld op: 01-01-1970',
+        type: NEARBY_TYPE,
+      },
+    ]
 
-    expect(get).not.toHaveBeenCalled()
     render(
       withAssetSelectContext(<DetailPanel {...props} />, {
         ...contextValue,
         selection,
       })
     )
 
-    expect(get).toHaveBeenCalledTimes(1)
-    expect(get).toHaveBeenCalledWith(
-      expect.stringContaining(configuration.GEOGRAPHY_PUBLIC_ENDPOINT)
-    )
+    expect(
+      screen.getByText(`${selection[0].description} - ${selection[0].label}`)
```

Comment:
dit valt geloof ik samen met de bug die we 9-6-22 tegenkwamen. 
We zouden de queries iets specifieker kunnen maken zoals 

    expect(
      screen.getByTestId('mockAssetList')
    ).toHaveTextContent(`${selection[0].description} - ${selection[0].label}`)

Reply:
Beter zo!

Reply:
aah fijn!